databases/week2/homework_part3.sql

```diff
@@ -0,0 +1,43 @@
+-- Part 3: More queries
+USE hyfdatabase;
+
+-- Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT
+    *
+FROM
+    task
+JOIN
+    user
+ON user.id = task.user_id
+WHERE user.email LIKE "%@spotify.com";
```

Comment:
General comment of query formatting, applicable to most of your queries. I suggest to follow this structure:
 
SELECT
    [COLUMN NAMES]
FROM [TABLE NAME]
JOIN [TABLE NAME + JOIN CONDITION]
    [EXTRA JOINS + CODITION]
WHERE [CONDITION]
GROUP BY [COLUMN NAMES]
ORDER BY [COLUMN NAMES]

Pointer: improves query readability and clarity, specially the FROM and JOIN shouldn't have line breaks in the middle of their statement.

Reply:
Ok. Super